src/components/scrollManager.js

```diff
@@ -532,9 +532,7 @@ import layoutManager from 'layoutManager';
 
     if (isEnabled()) {
         dom.addEventListener(window, 'focusin', function(e) {
-            setTimeout(function() {
```

Comment:
This is untested.

Reply:
Doesn't work.
There's a timeout https://github.com/calvellido/focus-options-polyfill/blob/c841df179f81bb81ee51309d9c6b77066cfad155/index.js#L74-L80
It was added to support iOS (13?) but now polyfill has "blinking".

I tried to remove timeout from polyfill and returned timeout in `scrollManager` (it is required in this case) - `scrollManager` starts to work (obviously), but `emby-scroller` still doesn't work.